psi4/driver/procrouting/proc.py

```diff
@@ -1402,8 +1399,8 @@ def scf_helper(name, post_scf=True, **kwargs):
 
     # Write out orbitals and basis; Can be disabled, e.g., for findif displacements
     if kwargs.get('write_orbitals', True):
-        fname = os.path.split(os.path.abspath(core.get_writer_file_prefix(scf_molecule.name())))[1]
-        write_filename = os.path.join(psi_scratch, fname + ".180.npy")
+        write_filename = scf_wfn.get_scratch_filename(180) + '.npy'
```

Comment:
I think `write_filename` should do the right thing as well. I did write that in I believe.

Reply:
I had to patch that one up.  The extra extension here is to make sure the `register_numpy_file` does the right thing.  I'll add a commit to fix that routine too.